zerver/openapi/zulip.yaml

```diff
@@ -2324,12 +2327,16 @@ paths:
                                     [
                                       {
                                         "type": "hotspot",
-                                        "name": "intro_streams",
-                                        "title": "Catch up on a stream",
-                                        "description": "Messages sent to a stream are seen by everyone subscribed to that stream. Try clicking on one of the stream links below.",
+                                        "name": "intro_compose",
+                                        "title": "Compose",
+                                        "description": "Click here to start a new conversation. Pick a topic (2-3 words is best), and give it a go!",
```

Comment:
Changing this here means that it won't have to be updated for the stream to channel rename.